content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,314 @@
 ---
 layout: "enterprise"
-page_title: "Logging - Infrastructure Administration - Terraform Enterprise"
+page_title: "Log Forwarding - Infrastructure Administration - Terraform Enterprise"
 ---
 
-# Terraform Enterprise Logs
+# Terraform Enterprise Log Forwarding
 
-This document contains information about interacting with Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
+Terraform Enterprise supports forwarding its logs to one or more external
+destinations, aptly referred to as log forwarding. This page details how to
+configure log forwarding in Terraform Enterprise.
 
-## Application Logs
+## Requirements
 
-Terraform Enterprise runs in a set of Docker containers. As such, any tooling that can interact with Docker logs can read the logs. This includes the command `docker logs`, as well as access via the [Docker API](https://docs.docker.com/engine/api/v1.36/#operation/ContainerLogs).
+Log forwarding requires:
 
-An example of a tool that can automatically pull logs for all docker containers is [logspout](https://github.com/gliderlabs/logspout).
-Logspout can be configured to take the Docker logs and send them to a syslog endpoint. Here's an example invocation:
+- Terraform Enterprise running on an instance with the `systemd-journald`
+service started and enabled. Execute `systemctl status systemd-journald` to
+check if the `systemd-journald` service is started and enabled.
+- A version of Docker that supports the `journald` logging driver. Execute
+`docker info --format '{{.Plugins.Log}}'` to check if the `journald` plugin is
+listed.
+- Network connectivity between Terraform Enterprise and the desired external
+destination(s) where logs should be forwarded to.
+
+## Configuration
+
+In order to use log forwarding, it must be enabled and configured. Once that is
+done, the Terraform Enterprise application will need to be restarted for the log
+forwarding configuration to take effect.
+
```

Comment:
I wonder if we can just delete this Configuration heading and move the others up to be at level 2. I say this because this is the last level 2 on the page, so there's no need to separate out this content from...other level 2 content if that makes sense. I think it would be faster and clearer to just remove "configuration" and just have the descriptive headers you have below.

This note about needing to restart is important and I also wonder if we need to make it more salient/move it somewhere else where the user is more likely to see it/not miss it. Thoughts?

Reply:
Love this! I removed the Configuration heading, moved the existing headings up to level 2, and added a Restart Terraform Enterprise heading.

Reply:
I like it!!! I did one tiny edit for you for style, but otherwise this is good, you can resolve!

Reply:
Accepted the edit! Thank you!